src/styles/stars.css

```diff
@@ -0,0 +1,14 @@
+.stars {
+  display: flex;
+  &__star {
+    background-image: url(/src/images/star.svg);
+    width: 16px;
+    height: 16px;
+    background-repeat: no-repeat;
+    background-position: center;
+    margin-right: 4px;
+  }
```

Comment:
Nested selectors in CSS should be avoided as they are not part of the standard CSS syntax. Consider refactoring this part using a preprocessor like SASS/SCSS if you want to use nesting.